intellij/install.sh

```diff
@@ -4,9 +4,10 @@
 bold=$(tput bold)
 normal=$(tput sgr0)
 
-# Try to find any intellij installation
+# Try to find any IntelliJ installation
 FOUND_INTELLIJ=false
-for i in $HOME/Library/Preferences/IntelliJIdea*  \
+for i in $HOME/Library/Application\ Support/JetBrains/IntelliJIdea* \
+         $HOME/Library/Preferences/IntelliJIdea* \
```

Comment:
I thought `$HOME/Library/Preferences/IntelliJIdea*` this was the macOS path? Did they change it again when they went from 2020.01 to 2020.1?

Reply:
I think it wasn't updated in #1